.env

```diff
@@ -0,0 +1,4 @@
+SECRET_WORDS='test test test test test test test test test test test junk'
```

Comment:
should this file named `.env.test`?

Reply:
combine `high-res.env` as well

Reply:
![image](https://github.com/KyberNetwork/kyberswap-interface/assets/132639843/cd538850-42da-427b-8ae5-63e3cd852784)
I've renamed the file to '.env.e2e'. Thanks